src/targetWidths.js

```diff
@@ -0,0 +1,109 @@
+// Taken from https://github.com/imgix/imgix.js/blob/eff194416a0efc9f0c528eb070466cd474ae0bf5/src/targetWidths.js
+
+function uniq(arr) {
+  var n = {},
+    r = [],
+    i;
+
+  for (i = 0; i < arr.length; i++) {
+    if (!n[arr[i]]) {
+      n[arr[i]] = true;
+      r.push(arr[i]);
+    }
+  }
+
+  return r;
+}
+
+import CONSTANTS from "./constants";
+
+const _screen = {
```

Comment:
Is this constant even used?

Reply:
No, I forgot to remove it when I did the update.